server/export_test.go

```diff
@@ -0,0 +1,6 @@
+package server
+
+var (
+	SetupLogger = setupLogger
```

Comment:
I think this is not needed, keep the tests with the package they are testing.  It is a go-ism to keep the tests near the code that is under test.

Reply:
resolved per https://github.com/brave/go-sync/pull/1#discussion_r429401836